report-summary-merged-prs

```diff
@@ -1404,6 +1423,8 @@ if __name__ == "__main__":
     MAGIC_COMMAND_FIND_MATERIL_BUDGET_COMPARISON_CHECKS = "TEST_FILE=" + JENKINS_ARTIFACTS_DIR + "/material-budget/RELEASE_NAME/ARCHITECTURE/comparison/Images/MBDiff.txt && test -f $TEST_FILE && grep '0$' $TEST_FILE | wc -l"
     MAGIC_COMMAND_FIND_VALGRIND = 'test -d ' + JENKINS_ARTIFACTS_DIR + '/valgrind/RELEASE_NAME'
     MAGIC_COMMAND_FIND_IGPROF = 'test -d ' + JENKINS_ARTIFACTS_DIR + '/igprof/RELEASE_NAME'
+    MAGIC_COMMAND_FIND_PROFILING_CHECKS_FILTER1 = 'test -f ' + JENKINS_ARTIFACTS_DIR + '/profiling/RELEASE_NAME/ARCHITECTURE/23434.21/step3_RAW2DIGI_L1Reco_RECO_RECOSIM_PU.resources.json'
+    MAGIC_COMMAND_FIND_PROFILING_CHECKS_FILTER2 = 'test -f ' + JENKINS_ARTIFACTS_DIR + '/igprof/RELEASE_NAME/ARCHITECTURE/profiling/23434.21/sorted_RES_CPU_step3.txt'
```

Comment:
just echo the file name to be saved in the json file e.g. see this https://github.com/cms-sw/cms-bot/blob/master/report-summary-merged-prs#L1402 . In your case you can echo "ARCHITECTURE/profiling/23434.21/sorted_RES_CPU_step3.txt/sorted_RES_CPU_step3.txt" and create a simple function to either see the result to "inprogress" or {"status": "passed", "data": "yourfile"}

you can then pass this function to find_general_test_results 

I can update this if this looks complicated to you